raiden/api/python.py

```diff
@@ -574,6 +574,58 @@ def _set_total_deposit_preconditions(
             )
             raise InsufficientFunds(msg)
 
+    def settle_light(
+        self,
+        registry_address: PaymentNetworkID,
+        token_address: TokenAddress,
+        creator_address: Address,
+        partner_address: Address,
+        signed_settle_tx: typing.SignedTransaction = None,
+    ):
+        """Settle a channel opened with `partner_address` for the given
+        `token_address`.
+        Race condition, this can fail if channel was settled externally.
+        """
+
+        channels_to_settle = ChannelValidator.validate_and_get_channels_to_settle(
+            token_address=token_address,
+            partner_addresses=[partner_address],
+            registry_address=registry_address,
+            raiden=self.raiden)
+
+        channel_ids = [channel_state.identifier for channel_state in channels_to_settle]
+
+        chain_state = views.state_from_raiden(self.raiden)
+
+        channel_state = views.get_channelstate_for(
+            chain_state=chain_state,
+            payment_network_id=registry_address,
+            token_address=token_address,
+            creator_address=creator_address,
+            partner_address=partner_address,
+        )
+
+        if channel_state.identifier not in channel_ids:
+            raise RaidenRecoverableError("Failed trying to settle a channel that's not in waiting_for_settle state")
+
+        channel_proxy = self.raiden.chain.payment_channel(
+            canonical_identifier=channel_state.canonical_identifier
+        )
+
+        channel_proxy.settle_channel_light(
+            block_identifier=views.state_from_raiden(self.raiden).block_hash,
+            signed_settle_tx=signed_settle_tx
+        )
+
+        waiting.wait_for_settle(
+            raiden=self.raiden,
+            payment_network_id=registry_address,
+            token_address=token_address,
+            channel_ids=channel_ids,
+            retry_timeout=DEFAULT_RETRY_TIMEOUT,
```

Comment:
wait_for_settle has retry_timeout as a required parameter, but it doesn't make sense to put a param here to let this be changed from outside since is not going to change for this case